src/prtfileemu.hpp

```diff
@@ -49,13 +49,10 @@ class PrintFileEmu {
     template<typename T> PrintFileEmu &operator<<(const T &x)
     {
         if(callback != nullptr){
+            // This drops formatting streaming, but shows up in unreal - Joe 7/7/2022.
```

Comment:
It sounds like you mean the old version was not showing up in Unreal? We should fix it, not get rid of the formatting. What happened in Unreal, were there no callbacks ever called or what was the contents of them?

Reply:
Will revert